### PR TITLE
Camel-case KYC settings, rename expiration date

### PR DESCRIPTION
**Pull request by subenson**

### CODE REVIEW

File: openapi/components/schemas/KycDocument/IdentityMatches.yaml [hunk 1]
```diff
@@ -28,6 +28,11 @@ properties:
     description: The expiry date found on the document, null if not found.
     type: string
     format: date-time
+    deprecated: true
+  expirationDate:
+    description: The expiry date found on the document, null if not found.
```
Comment (ashkarpetin) [inline openapi/components/schemas/KycDocument/IdentityMatches.yaml:33]:
```suggestion
    description: The expiration date found on the document, null if not found.
```

File: openapi/components/schemas/Organizations/OrganizationSettings.yaml [hunk 2]
```diff
@@ -21,7 +21,7 @@ properties:
     type: object
     description: The settings for tax calculation.
```
Comment (adamaltman) [inline openapi/components/schemas/Organizations/OrganizationSettings.yaml:22]: Tax calculation?

Comment (subenson) [reply openapi/components/schemas/Organizations/OrganizationSettings.yaml:22]:
```suggestion
    description: The settings for KYC scoring.
```

**Review by ashkarpetin — DISMISSED** [review]

**Review by vykhrystiuk — DISMISSED** [review]

File: openapi/components/schemas/KycDocument/Settings/KycSettingsIdentity.yaml [hunk 3]
```diff
@@ -29,11 +29,11 @@ properties:
         minimum: 0
         maximum: 100
         description: The weight of lastName.
-      expiryDate:
+      expirationDate:
         type: integer
         minimum: 0
         maximum: 100
-        description: The weight of expiryDate.
+        description: The weight of expirationDate.
```
Comment (adamaltman) [inline openapi/components/schemas/KycDocument/Settings/KycSettingsIdentity.yaml:36]:
```suggestion
        description: The weight of `expirationDate`.
```

**Review by ashkarpetin — APPROVED** [review]

**Review by vykhrystiuk — APPROVED** [review]

**Pull request merged**